### PR TITLE
fix: Qwen3.5 RMSNorm weight convention mismatch causing endless spaces

### PR DESCRIPTION
**Pull request by co-esquie**
# Fix Qwen3.5 RMSNorm weight loading
## Summary

After implementing https://github.com/ericcurtin/inferrs/pull/146 I've tested more dense models and ran into issues with some of them

Qwen3.5 models produce degenerate output (endless whitespace) because RMSNorm weights are loaded with the wrong convention. The PyTorch `Qwen3_5RMSNorm` stores **residual weights** (initialized at `torch.zeros`, forward computes `(1.0 + weight) * normalize(x)`) but `candle_nn::rms_norm()` treats them as **direct multipliers** (`weight * normalize(x)`), producing near-zero activations throughout the network.

## Details

The Qwen3.5 architecture introduces a custom `Qwen3_5RMSNorm` class that differs from the standard `RMSNorm` used by Qwen2/3:

| | Standard RMSNorm | Qwen3.5 RMSNorm |
|---|---|---|
| Init | `torch.ones(dim)` | `torch.zeros(dim)` |
| Checkpoint values | ~1.0 | ~0.0 (residual from 1.0) |
| Forward | `weight * normalize(x)` | `(1.0 + weight) * normalize(x)` |

When `candle_nn::rms_norm()` loads the Qwen3.5 checkpoint weights (~0.0) and uses them as direct multipliers, every norm layer outputs near-zero values. The model collapses to predicting the same token repeatedly (a whitespace token), producing endless spaces with no stop condition.

## Fix

Add `rms_norm_with_offset()` helper that:
1. Loads the checkpoint weight via `VarBuilder`
2. Adds the offset (1.0) to convert residuals into direct multipliers
3. Constructs `RmsNorm::new(adjusted_weight, eps)`

This is applied to all 5 norm-layer types that use the Qwen3.5 convention:

- `input_layernorm` (all layers)
- `post_attention_layernorm` (all layers)
- `norm` (final layer norm)
- `q_norm` (full-attention layers)
- `k_norm` (full-attention layers)

The `LinearAttn` gated norm is **unaffected** — it uses `rms_norm_tensor()` with weights stored near 1.0 (correct convention).

## Testing

```
inferrs run Qwen/Qwen3.5-2B "What is the capital of France?"
```

Before fix: endless whitespace output, model never stops.
After fix: correct, coherent response.

### Last word

Despite the prefill performances (I'm looking into [Blelloch Algorithm](https://medium.com/nerd-for-tech/understanding-implementation-of-work-efficient-parallel-prefix-scan-cca2d5335c9b) and [GatedDeltaNet](https://github.com/NVlabs/GatedDeltaNet)) qwen 3.5 dense models should be stable

### CODE REVIEW

File: inferrs/src/models/qwen3_5.rs [hunk 1]
```diff
@@ -18,6 +18,13 @@ use crate::models::attention_utils::{
 };
 use crate::turbo_quant::{TurboQuantConfig, TurboQuantKvCache};
 
+fn rms_norm_with_offset(size: usize, eps: f64, vb: VarBuilder, offset: f64) -> Result<RmsNorm> {
+    let weight = vb.get_with_hints(size, "weight", Init::Const(1.0))?;
+    let ones = Tensor::ones(weight.shape(), weight.dtype(), weight.device())?;
+    let adjusted = ((&ones * offset)? + &weight)?;
+    Ok(RmsNorm::new(adjusted, eps))
+}
```
[the remark this answers is not in the document]
Comment (co-esquie) [reply inferrs/src/models/qwen3_5.rs:25]: Good one, implemented

File: inferrs/src/models/qwen3_5.rs [hunk 2]
```diff
@@ -18,6 +18,13 @@ use crate::models::attention_utils::{
 };
 use crate::turbo_quant::{TurboQuantConfig, TurboQuantKvCache};
 
+fn rms_norm_with_offset(size: usize, eps: f64, vb: VarBuilder, offset: f64) -> Result<RmsNorm> {
+    let weight = vb.get_with_hints(size, "weight", Init::Const(1.0))?;
```
[the remark this answers is not in the document]
Comment (co-esquie) [reply inferrs/src/models/qwen3_5.rs]: Same as gemini

**Comment by co-esquie** [comment]
All bots feedbacks applied
